examples/notebooks/Cifar100_bench_amp.ipynb

```diff
@@ -45,7 +45,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "2) Let's install Nvidia/Apex package:"
+    "2) Let's install Nvidia/Apex package:\n",
```

Comment:
I'm not sure if repeating twice the same command makes sense. Take a look at the rendering : 
<img width="902" alt="Screen Shot 2020-12-28 at 21 30 51" src="https://user-images.githubusercontent.com/2459423/103241628-17c81080-4954-11eb-8795-fbae07ff3a1b.png">


What do you think ?

Reply:
You are right. I prefer keep the code cell. Ok ?

Reply:
Sounds good!